app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The class attribute `alive` should be accessed using the class name `Animal` in the `__init__` method to adhere to the checklist item: 'Use the class name to get access to the class attributes'.